web/html/src/manager/maintenance/calendar/web-calendar.tsx

```diff
@@ -0,0 +1,262 @@
+import * as React from "react";
+import { useState, useEffect, useRef } from "react";
+
+import FullCalendar from "@fullcalendar/react";
+import dayGridPlugin from "@fullcalendar/daygrid";
+import timeGridPlugin from "@fullcalendar/timegrid";
+import interactionPlugin from "@fullcalendar/interaction";
+import allLocales from "@fullcalendar/core/locales-all";
```

Comment:
While this isn't a huge consideration with Uyuni, this might add a lot of volume to the package while we only support a small number of locales. I don't know if we currently have a way to manage those in the frontend packaging? Perhaps @ncounter or @cbbayburt know?

Reply:
I am aware if this. AFAIK there is also the option to only import the locales that we support. Downside of that is that we would have to revisit this whenever we decide to support a new locale.

Reply:
I would not see a bid deal to define one by one, if we add more it will not be dynamically added I guess, but we will have to add somewhere else the enablement. So, I'm fine with both approaches. If we go with the specifying one by one, I'd love to see one single point to configure it, instead of having it inside of a component, then maybe in another one, and then in Java backend, and maybe more. Question: how many locales we support now? Btw, it's okay if we keep them all for now, but take a note/issue to revisit maybe later.

Reply:
Sounds good I will add a point about it to the issue.